app/uk/gov/hmrc/cataloguefrontend/vulnerabilities/VulnerabilitiesConnector.scala

```diff
@@ -54,4 +57,16 @@ class VulnerabilitiesConnector @Inject() (
       .get(url"$url/vulnerabilities/api/vulnerabilities/counts?service=$service&team=$team&environment=${environment.map(_.asString)}")
       .execute[Seq[TotalVulnerabilityCount]]
   }
+
+  def timelineCounts(service: Option[String], team: Option[String], vulnerability: Option[String], curationStatus: Option[String], from: LocalDate, to: LocalDate)(implicit hc: HeaderCarrier): Future[Seq[VulnerabilitiesTimelineCount]] = {
+    implicit val stcr: Reads[VulnerabilitiesTimelineCount] = VulnerabilitiesTimelineCount.reads
+    val fromInstant = DateTimeFormatter.ISO_INSTANT.format(from.atStartOfDay().toInstant(ZoneOffset.UTC))
+    val toInstant   = DateTimeFormatter.ISO_INSTANT.format(to.atStartOfDay().toInstant(ZoneOffset.UTC))
```

Comment:
CLARIFICATION: should that be at the end of the day?

Reply:
Yep - have fixed (note there is no native endOfDay function)